src/UI/Implementation/ComponentRendererFSLoader.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/* Copyright (c) 2017 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation;
```

Comment:
Are you sure to put all those files directly into implementation? To me they belong into Renderer? Not sure though.